java/pom.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.example</groupId>
+    <artifactId>GraphBulkExecutor</artifactId>
+    <version>1.0</version>
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <cosmosSDKVersion>4.24.0</cosmosSDKVersion>
+        <lombokVersion>1.18.22</lombokVersion>
+        <commonsVersion>3.12.0</commonsVersion>
+        <snapshotDependencyAllowed>true</snapshotDependencyAllowed>
+        <skipIntegrationTests>true</skipIntegrationTests>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
```

Comment:
If we can avoid usage of lombok, we should. Reason being it has caused some dependency issues for customers in the past. We don't want customers to fall into dependency issues when using this sample, as customers would directly copy paste this.

Reply:
Removed the dependency on lombak. Needed quite a bit of boiler plate bits added to replace the functionality.